templates/privacy-preferences-modal.php

```diff
@@ -73,7 +73,7 @@
 											<p><?php echo esc_html( $consent_type['name'] ); ?></p>
 											<?php if ( $consent_type['policy-page'] ) : ?>
 												<span class="gdpr-always-active"><?php esc_html_e( 'Required', 'gdpr' ); ?></span>
-												<input type="hidden" name="user_consents[]" value="<?php echo esc_attr( $consent_key ); ?>" checked style="display:none;">
+												<input type="hidden" name="user_consents[]" value="<?php echo esc_attr( $consent_key ); ?>" style="display:none;">
```

Comment:
We're no longer checking by default, just checking (hehe) that's what we intended?

Reply:
That's a good joke.

You can only `checked` a radio or checkbox field, so this shouldn't be doing anything.

Reply:
ha, I didn't even spot the input type 👍